airbyte-integrations/connectors/source-file/source_file/source.py

```diff
@@ -292,7 +339,9 @@ def parse_file(logger, reader_format: str, url, reader_options: dict) -> List:
             # pandas.read_csv additional arguments can be passed to customize how to parse csv.
             # see https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html
             result.append(pd.read_csv(url, **reader_options))
-        elif reader_format == "json":
+        elif reader_format == "flat_json":
```

Comment:
> flat_json where the provided JSON is considered already flat and doesn't try to infer deeper JSON schemas (processed as strings if there is further json objects)

what's the use case for flat_json? why wouldn't we want to just treat json like json?

Reply:
It's using two methods of loading json data, since it's not working so well (through the UI) i haven't been able to choose one way or the other yet... I'll disable the flat json option for now in the UI